server/src/test/java/org/elasticsearch/common/logging/LoggersTests.java

```diff
@@ -25,7 +26,8 @@
 public class LoggersTests extends ESTestCase {
 
     public void testParameterizedMessageLambda() throws Exception {
-        final MockAppender appender = new MockAppender("trace_appender");
+        // adding a random id to allow test to run multiple times. See AbstractConfiguration#addAppender
+        final MockAppender appender = new MockAppender("trace_appender" + UUID.randomUUID());
```

Comment:
I don’t think a uuid is necessary. It also would not reproduce. Instead you can use a random string or number using the seeded random of the test?

Reply:
good point. will do